harness/determined/pytorch/_reducer.py

```diff
@@ -282,6 +285,28 @@ def reset_reducers(self) -> None:
         for wrapped in self._wrapped_reducers:
             wrapped.reset()
 
+    # When wrap_reducer is passed a Callable, it returns a _SimpleReducer.
+    @overload
```

Comment:
Neat; I was not aware of this machinery.

Reply:
neither was I, it was @felix-schneider who [explained](https://github.com/determined-ai/determined/issues/2926) it to me.